src/main/kotlin/org/openbroker/common/model/BankAccount.kt

```diff
@@ -1,10 +1,10 @@
-package org.openbroker.se.model
+package org.openbroker.common.model
 
 private val clearingNumberRegex = Regex("^[1-9][0-9]{3}\$")
 private val accountNumberRegex = Regex("^[0-9]{1,11}$")
 
 /**
- * A Swedish bank account consisting of a clearing number and an
+ * A bank account consisting of a clearing number and an
```

Comment:
Is the regex above applicable for Norwegian account and clearing numbers as well? I think we should consider creating a separate BankAccount class for each country if that is the case.

Reply:
Well, yes it is applicable. But thinking closer upon this i still think you're correct. It's an object which "feels" like it's matching in this instance, but not as a rule.
I will adjust the structure and move BankAccount from 'common' to both 'no' and 'se'.